index.html

```diff
@@ -58,32 +50,29 @@
 
         const $ = {};
         window.$ = $;
+
+        // Create client configuration.
+        const configBuilder = Nimiq.Client.Configuration.builder();
         if (clientType === 'light') {
-            $.consensus = await Nimiq.Consensus.light();
-        } else if (clientType === 'nano') {
-            $.consensus = await Nimiq.Consensus.nano();
+            // Adding the Mempool feature triggers a light consensus.
+            configBuilder.feature(Nimiq.Client.Feature.MEMPOOL);
         }
 
-        $.blockchain = $.consensus.blockchain;
-        $.mempool = $.consensus.mempool;
-        $.network = $.consensus.network;
+        // Instantiate the client.
```

Comment:
```suggestion
        // create client based on current configuration; the client will automatically connect to the network.
```
It does connect automatically, correct? (`connect()` removed below)